### PR TITLE
Turn exercise 026 to practice

### PR DESCRIPTION
**Pull request by Ozyugoo**

### CODE REVIEW

**Comment by Ozyugoo** [comment]
@cuihtlauac, Kindly check this, it compiled ok.

File: practice/026/answer/impl.ml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+let rec extract k list =
+  if k <= 0 then [[]]
+  else match list with
+       | [] -> []
+       | h :: tl ->
+          let with_h = List.map (fun l -> h :: l) (extract (k - 1) tl) in
+          let without_h = extract k tl in
+          with_h @ without_h
```
Comment (cuihtlauac) [inline practice/026/answer/impl.ml]:
```suggestion
          with_h @ without_h

```

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: practice/026/answer/impl.ml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
```
Comment (divyankachaudhari) [inline practice/026/answer/impl.ml:9]: Please add a newline at the end of the file :)

Comment (Ozyugoo) [inline practice/026/answer/impl.ml]: It does have a newline @divyankachaudhari. Thank you

Comment (cuihtlauac) [reply practice/026/answer/impl.ml]: Your are not using the right tool to view and edit:

Here is the end of your file using `hexdump -C`
```
000000d0  65 78 74 72 61 63 74 20  6b 20 74 6c 20 69 6e 0a  |extract k tl in.|
000000e0  20 20 20 20 20 20 20 20  20 20 77 69 74 68 5f 68  |          with_h|
000000f0  20 40 20 77 69 74 68 6f  75 74 5f 68 0a 20 20 20  | @ without_h.   |
00000100  20 20 20 20 20 20 20
```

As you can see, there are plenty of space characters (`20`) and no newline (`0a`). This is not good. After the meaningful text (here `75 74 5f 68`), you should only have a newline and nothing after.

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

**Comment by Ozyugoo** [comment]
Thank you @cuihtlauac, please is this an improvement.

**Review by cuihtlauac — APPROVED** [review]

**Pull request merged**